addons/udes_priorities/models/stock_move.py

```diff
@@ -39,4 +43,33 @@ def _priority_domain(self):
     def get_priorities_for_selection(self):
         Priorities = self.env["udes_priorities.priority"]
         priorities = Priorities.search(self._priority_domain())
+
+        # hard coded default value means there is always a priority to set
+        normal = self.env.ref("udes_priorities.normal")
+        priorities |= normal
```

Comment:
Do we want to do this only if `priorities` is empty?

Reply:
resolving as per comments